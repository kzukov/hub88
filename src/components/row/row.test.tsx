import React from "react";
import Row from "./row";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("<Row />", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Row>children</Row>);

    expect(asFragment()).toMatchSnapshot();
  });
});
