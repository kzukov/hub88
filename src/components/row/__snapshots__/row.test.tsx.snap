// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="row"
  >
    children
  </div>
</DocumentFragment>
`;
